### PR TITLE
Fix the error thrown by horizontal-vector-heatmap

### PR DESCRIPTION
**Pull request by pkerpedjiev**
## Description

What was changed in this pull request?

`horizontal-vector_heatmap` tracks were throwing an error when used with `vector` data from `bigwig` and `bigbed` data sources. This was because they were returning data smaller than the expected tile size. This fix padds their return data to make sure it matches the specified `tile_size`.

Fixes #___

## Checklist

- [x] Unit tests added or updated
- [o] Documentation added or updated
- [o] Example added or updated
- [o] Screenshot for visual changes (e.g. new tracks)
- [x] Updated CHANGELOG.md

### CODE REVIEW

File: app/scripts/HeatmapTiledPixiTrack.js [hunk 1]
```diff
@@ -1026,6 +1026,19 @@ class HeatmapTiledPixiTrack extends TiledPixiTrack {
 
     this.renderingTiles.add(tile.tileId);
 
+    if (this.tilesetInfo.tile_size) {
+      if (tile.tileData.dense.length < this.tilesetInfo.tile_size) {
```
Comment (flekschas) [inline app/scripts/HeatmapTiledPixiTrack.js]: Does that mean the server can send corrupt data? I like this fix but shouldn't we make sure the server sends proper data too? (Maybe there's another fix?)

File: test/HorizontalMultivecTests.js [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+      done,
+      {
+        style: 'width:800px; height:400px; background-color: lightgreen',
+        bounded: true,
+      })
+    );
+  });
+
+  it('not have errors in the loaded viewconf', (done) => {
+    done();
```
Comment (flekschas) [inline test/HorizontalMultivecTests.js]: Where is the test? Am I missing something?

File: app/scripts/HeatmapTiledPixiTrack.js (hunk 1 above)
Comment (pkerpedjiev) [reply app/scripts/HeatmapTiledPixiTrack.js]: It's not exactly corrupt. It's just not complete. I think it's fine if the server sends a truncated array if the remainder is supposed to be NaN.

File: test/HorizontalMultivecTests.js (hunk 2 above)
Comment (pkerpedjiev) [reply test/HorizontalMultivecTests.js]: The test is just loading that viewconf without throwing an error.

Comment (pkerpedjiev) [reply test/HorizontalMultivecTests.js]: Is there a way to explicitly test for no error?

**Pull request merged**